src/org-manager/store/effects/org-pending.effects.ts

```diff
@@ -56,7 +56,8 @@ export class PendingOrgEffects {
       organisation.status = apiOrg.status;
       organisation.admin = `${apiOrg.superUser.firstName} ${apiOrg.superUser.lastName}`;
       organisation.dxNumber = apiOrg.contactInformation[0].dxAddress;
-      organisation.address = `${apiOrg.contactInformation[0].addressLine1}, ${apiOrg.contactInformation[0].county},
+      organisation.address = `${apiOrg.contactInformation[0].addressLine1},
```

Comment:
u need to move all of this mapOrganisation in the reducer rather than have it with the effect. also you could move it in AppUtil and unit test it there and  then imported into the reducer

Reply:
ok, agreed. Will do that